README.md

```diff
@@ -1,3 +1,10 @@
+[![npm](https://img.shields.io/npm/v/@first-lego-league/ms-auth.svg)](https://www.npmjs.com/package/@first-lego-league/ms-auth)
```

Comment:
That are these?

Reply:
badges, you can see a preview in ms-correlation